src/Package/Impl/DataInspect/VariableView.xaml.cs

```diff
@@ -159,7 +159,7 @@ public partial class VariableView : UserControl, IDisposable {
         private Task DeleteCurrentVariableAsync() {
             var node = RootTreeGrid.SelectedItem as ObservableTreeNode;
             var ew = node?.Model?.Content as VariableViewModel;
-            return ew != null ? ew.DeleteAsync() : Task.CompletedTask;
```

Comment:
Don't change this - awaiting code doesn't expect this method to return `null`. `null` is allowed only if method is `async`.